packages/FormElement/src/FormElement.js

```diff
@@ -66,117 +70,91 @@ function FormElement(props) {
     isDisabled,
     isInline,
     isLabelVisuallyHidden,
-    isReadOnly,
     label,
     size,
     ...moreProps
   } = props;
 
   const extractedChildren = extractChildren(children, [
-    "FormElement.Description",
-    "FormElement.Error",
-    "FormElement.Help",
-    "FormElement.Instructions",
+    subComponentDisplayNames.Description,
+    subComponentDisplayNames.Error,
+    subComponentDisplayNames.Help,
+    subComponentDisplayNames.Instructions,
+    subComponentDisplayNames.Content,
   ]);
   const ariaDescriptionId = React.useRef(uuidv4()).current;
   const ariaErrorId = React.useRef(uuidv4()).current;
   const ariaInstructionsId = React.useRef(uuidv4()).current;
-  const hasError = !!extractedChildren["FormElement.Error"] && !!extractedChildren["FormElement.Error"].props.children;
   const uniqueInputId = React.useRef(uuidv4()).current;
-  const inputId = isNil(id) || id === "" ? uniqueInputId : id;
+  const hasError =
+    !!extractedChildren[subComponentDisplayNames.Error] &&
+    !!extractedChildren[subComponentDisplayNames.Error].props.children;
+  const idForLabel = isNil(id) || id === "" ? uniqueInputId : id;
```

Comment:
maybe also rename `uniqueInputId ` to `uniquLabelId `

Reply:
It's not an id on the Label though, its the id of the input that gets used by the `htmlFor` attribute on the `label` element